src/Apps/Home/Components/HomeFeaturedMarketNews.tsx

```diff
@@ -125,7 +125,7 @@ const HomeFeaturedMarketNewsContainer: React.FC = ({ children }) => {
   return (
     <>
       <Flex justifyContent="space-between" alignItems="center">
-        <Text variant="xl">Market News</Text>
+        <Text variant="xl">Artsy Editorial</Text>
```

Comment:
**question(non blocking):** I am not 100% sure what is the preferred way to add strings. When should we add it on the i18n file and when not, will open a thread on #practice-web to check!

Reply:
left a question [here](https://artsy.slack.com/archives/C2TQ4PT8R/p1673527351037269) to know what to do for the next copy changes that we will encounter